### PR TITLE
Merge esri-leaflet-2.0.0 into master

### PR DESCRIPTION
**Pull request by kneemer**
Ok, I did a manual merge of esri-leaflet-2.0.0 into master, resolved the conflicts and pushed it up as master-merge.  The src code looks as expected and the tests pass.  If you could give this a good look as well @jgravois we can move forward.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,4 +39,7 @@ dist/**/*.map
 dist/**/*.css
 dist/**/*.gif
 dist/**/*.png
-dist/**/*.jpg
\ No newline at end of file
+dist/**/*.jpg
+
+
+npm-debug.log
```
Comment (jgravois) [inline .gitignore]: not a huge deal, but looks like `npm-debug.log` snuck in twice.

File: README.md [hunk 2]
```diff
@@ -50,13 +50,17 @@ You can also find a side by side comparison of the ArcGIS API for JavaScript [he
...
+
+### Limitations
+* [Simple Marker](http://resources.arcgis.com/en/help/arcgis-rest-api/02r3/02r3000000n5000000.htm#GUID-C8D40B32-5F4B-45EB-8048-6D5A8763E13B) symbols do not support rotation (ie: the 'angle' property is ignored).
+* Polygons only support [solid fill](http://resources.arcgis.com/en/help/arcgis-rest-api/02r3/02r3000000n5000000.htm#GUID-517D9B3F-DF13-4E79-9B58-A0D24C5E4994).  This does not include advanced fill types like PictureFill, Backward Diagonal, DiagonalCross, etc.
+* [Text](http://resources.arcgis.com/en/help/arcgis-rest-api/02r3/02r3000000n5000000.htm#ESRI_SECTION1_94E8CE0A9F614ABC8BEDDBCB0E9DC53A) symbols are not supported.
+* [Unique value](http://resources.arcgis.com/en/help/arcgis-rest-api/index.html#//02r30000019t000000#GUID-35C6482C-9BB2-4839-A180-25DCF62CB504) renderers based on more than a single field are not supported.
 
 ### Limitations
 * [Simple Marker](http://resources.arcgis.com/en/help/arcgis-rest-api/02r3/02r3000000n5000000.htm#GUID-C8D40B32-5F4B-45EB-8048-6D5A8763E13B) symbols do not support rotation (ie: the 'angle' property is ignored).
```
Comment (jgravois) [inline README.md]: 'Limitations' are duplicated too.

**Comment by kneemer** [comment]
Thanks for catching those, I was paying closer attention to everything in src and just scanned everything else.

**Comment by jgravois** [comment]
awesome work!

**Pull request merged**